zstacklib/zstacklib/network/ovs/dpdk.py

```
@@ -0,0 +1,626 @@
+
+            if 'dpdkvdpa' in nic_type:
+                options = 'vdpa-socket-path'
+            elif 'dpdkvhostuserclient' in nic_type:
+                options = 'vhost-server-path'
+            else:
+                return sock_path
+
+            sock_path = shell.call(
+                CTL_BIN + f'get interface {nic_name} options:{options}'
+            ).strip().strip('"')
+            return sock_path
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)